Source/control.cpp

```diff
@@ -310,6 +310,36 @@ void SetSpellTrans(spell_type t)
 	}
 }
 
+void PrintSBookSpellType(const Surface &out, Point position, const std::string &text, uint8_t rectColorIndex)
+{
+	Point rect {position};
+	rect += Displacement { 0, -SPLICONLENGTH + 1};
+
+	// Top
+	DrawHorizontalLine(out, rect, SPLICONLENGTH, rectColorIndex);
+	DrawHorizontalLine(out, rect + Displacement {0, 1}, SPLICONLENGTH, rectColorIndex);
+
+	// Bottom
+	DrawHorizontalLine(out, rect + Displacement {0, SPLICONLENGTH - 2}, SPLICONLENGTH, rectColorIndex);
+	DrawHorizontalLine(out, rect + Displacement {0, SPLICONLENGTH - 1}, SPLICONLENGTH, rectColorIndex);
+
+	// Left Side
+	DrawVerticalLine(out, rect, SPLICONLENGTH, rectColorIndex);
+	DrawVerticalLine(out, rect + Displacement {1, 0}, SPLICONLENGTH, rectColorIndex);

```

Comment:
Spacing around curly braces are wrong in a lot of places, run clang-format to fix them all automatically.
```suggestion
	DrawVerticalLine(out, rect + Displacement { 1, 0 }, SPLICONLENGTH, rectColorIndex);
```